### PR TITLE
Bubble Sort - Ruby's 1st Project

### PR DESCRIPTION
**Pull request by camfilho**

### CODE REVIEW

File: bubblesort.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+def bubble_sort(arr2)
+  return arr2 unless arr2.is_a? Array
+
+  arr = arr2.dup
+  sorted = false
+  until sorted
+    sorted = true
+    (0...arr.length - 1).each do |i|
+      # swaping
```
Comment (whiz25) [inline bubblesort.rb]: remove this comment

File: bubblesort.rb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      end
+    end
+  end
+  arr
+end
+
+def bubble_sort_by(arr2)
+  return arr2 unless arr2.is_a? Array
+
+  arr = _arr2.dup
```
Comment (whiz25) [inline bubblesort.rb]: is it ```_arr2``` or ```arr2```?. Undefined error is being thrown on that line.

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Amazing job 👍 
Your code is clean. That's good. However, we can not merge this PR yet.
- [ ] check my comments and implement those changes
- [ ] make your README more descriptive. Add a section for ```Getting started, stacks used, etc```.

#### Make those changes and re-submit for another review.

### Martin Afani

**Review by Torres-ssf — APPROVED** [review]
### Project Approved :+1: :tada: 

I'm approving this project, nice job on it
Go on and merge to master.

Good luck on the next ones.
Happy coding :computer:

**Pull request merged**